features/step_definitions/official_copy_legal_txt.rb

```diff
@@ -0,0 +1,53 @@
+When(/^I attempt to view the Official Copy of the Register for a title$/) do
+	insert_title_non_private_individual_owner
+	visit_title_register_pdf(@title[:title_number])
+	convert_pdf_to_page
+end
+
+Then(/^I can see the edition dates in the correct format$/) do
+  expect(@pdf_pages[0]).to include('20 December 2015')
+end
+
+Then(/^I can see the dates of the entries on the register in the correct format$/) do
+	expect(@pdf_pages[0]).to include('6 November 1995')
+	expect(@pdf_pages[0]).to include('28 August 2014')
+	expect(@pdf_pages[0]).to include('1 July 2996')
+	expect(@pdf_pages[0]).to include('5 September 2005')
+end
+
+Then(/^I can see the class of the title in the proprietorship register$/) do
+	expect(@pdf_pages[0]).to include('Absolute')
```

Comment:
Can we check for a longer string, e.g. `Class: Absolute` or sth similar, so that we know this word wasn't a part of the document by accident?